CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## Unreleased
 
+- Added support for phoenix_html `v4`. You can also continue using phoenix_html `v3`.
```

Comment:
Maybe we can also document the newly added component for raw inputs (whatever name you choose)

Reply:
Is this enough [`ca1ec04` (#130)](https://github.com/bitcrowd/bitstyles_phoenix/pull/130/commits/ca1ec046c77efb6d5c05856eadcb80e988e89afb) ? I was struggling to find the right words to describe the components in the changelog and gave up on writing more, hoping people would just look for more details in the components' documentation

Reply:
Perfect!